src/firebase/api/experiments.js

```diff
@@ -0,0 +1,142 @@
+import firestore from '../firebase.js';
```

Comment:
This import is wrong, it should be `{ firestore }`. You're actually import `firebase` with this and renaming it to `firestore`

Reply:
ah duh! Thanks for spotting it!